scenarios/contact-menu.json

```diff
@@ -68,6 +68,50 @@
       "url": "{url}/civicrm/group?reset=1",
       "onReadyScript": "contact/manage-groups.js"
     },
+    {
+      "label": "Manage Groups - Edit Group settings modal",
+      "url": "{url}/civicrm/group?reset=1",
+      "selectors": [".ui-dialog"],
+      "onReadyScript": "contact/manage-groups-settings-modal.js"
```

Comment:
why are any script related to managing groups inside `contact/`? do the script have anything to do with a contact? Shouldn't there rather be a `manage-groups/` or `group/` folder?

Reply:
Since it was a part of contact menu I used contact folder. We can have another folder for this.